### PR TITLE
[Update] Replace instances of `AGSServiceFeatureTable.applyEdits` , 1st PR

### PR DESCRIPTION
**Pull request by yo1995**
## Description

This PR replaces various usage of `applyEdits` method on `AGSServiceFeatureTable` to `AGSServiceGeodatabase`, to prevent data inconsistency issue.

I'll break down the Epic into 2 PRs to make it easier to review. This is the 1st PR.

All the behaviors should remain the same. Coding style has been updated/refactored to follow our latest style guide. Also, a few UI related bugs are fixed. But we'll still request Sarat's review to ensure nothing is breaking the tests.

Also, all comment strings are formatted.

Finally, READMEs will be updated in yet another separate PR, after we finish updating the common designs.

## Linked Issue(s)

`common-samples/issues/3371`

## How To Test

Go though all changed samples and see if they work as expected.

<details><summary>Example change</summary>

```swift
/// Load and set a service geodatabase from a feature service URL.
/// - Parameter serviceURL: The URL to the feature service.
func loadServiceGeodatabase(from serviceURL: URL) {
    let serviceGeodatabase = AGSServiceGeodatabase(url: serviceURL)
    serviceGeodatabase.load { [weak self] error in
        guard let self = self else { return }
        if let error = error {
            self.presentAlert(error: error)
        } else {
            let featureTable = serviceGeodatabase.table(withLayerID: 0)!
            self.featureTable = featureTable
            self.serviceGeodatabase = serviceGeodatabase
            // Add the feature layer to the operational layers on map.
            let featureLayer = AGSFeatureLayer(featureTable: featureTable)
            self.mapView.map?.operationalLayers.add(featureLayer)
        }
    }
}

// then...
serviceGeodatabase.applyEdits { [weak self] (featureTableEditResults: [AGSFeatureTableEditResult]?, error: Error?) in
    if let error = error {
        self?.presentAlert(message: "Error while applying edits:\(error.localizedDescription)")
    } else {
        if let featureTableEditResults = featureTableEditResults,
           featureTableEditResults.first?.editResults.first?.completedWithErrors == false {
            self?.presentAlert(message: "Edits applied successfully")
        }
    }
}
```

</details>

### CODE REVIEW

File: arcgis-ios-sdk-samples/Edit data/Update geometries (feature service)/EditGeometryViewController.swift [hunk 1]
```diff
@@ -64,91 +95,89 @@ class EditGeometryViewController: UIViewController, AGSGeoViewTouchDelegate, AGS
...
-        // disable sketch editor
-        self.mapView.sketchEditor?.stop()
-        
-        // clear sketch editor
-        self.mapView.sketchEditor?.clearGeometry()
+        // Hide toolbar.
+        setToolbarVisibility(visible: false)
+        // Stop and clear sketch editor.
+        mapView.sketchEditor?.stop()
+        mapView.sketchEditor = nil
```
Comment (yo1995) [inline arcgis-ios-sdk-samples/Edit data/Update geometries (feature service)/EditGeometryViewController.swift:181]: Actually from reading another sample, I don't think it's needed to nullify the sketch editor every time. Can just create once in mapView didSet.

**Review by vquach2404 — APPROVED** [review]
Looks good!

File: arcgis-ios-sdk-samples/Edit data/Add features (feature service)/AddFeaturesViewController.swift [hunk 2]
```diff
@@ -16,81 +16,94 @@ import UIKit
...
-                    self?.presentAlert(message: "Edits applied successfully")
-                }
-                UIApplication.shared.hideProgressHUD()
+        guard serviceGeodatabase.hasLocalEdits() else { return }
+        serviceGeodatabase.applyEdits { [weak self] featureTableEditResults, error in
+            if let featureTableEditResults = featureTableEditResults,
+               featureTableEditResults.first?.editResults.first?.completedWithErrors == false {
+            } else if let error = error {
+                self?.presentAlert(message: "Error while applying edits: \(error.localizedDescription)")
             }
```
Comment (philium) [inline arcgis-ios-sdk-samples/Edit data/Add features (feature service)/AddFeaturesViewController.swift:99]: It seems odd to have an empty clause. Is it enough to check the error?

```suggestion
        serviceGeodatabase.applyEdits { [weak self] _, error in
            if let error = error {
                self?.presentAlert(message: "Error while applying edits: \(error.localizedDescription)")
            }
```

Comment (yo1995) [reply arcgis-ios-sdk-samples/Edit data/Add features (feature service)/AddFeaturesViewController.swift:99]: Oof, I thought linter could catch this sort of mistake 🤦

**Review by philium — APPROVED** [review]

**Review by saratchandrakarumuri — APPROVED** [review]
All updated samples are working without any issues

**Pull request merged**